cddl/example/psa-token.diag

```diff
@@ -1,5 +1,5 @@
 {
-  / psa-profile /                 -75000: "PSA_IOT_PROFILE_1",
+  / psa-profile /                     18: "http://arm.com/psa/2",
```

Comment:
drop psa- prefix in comment. Also for 26 & 28 below

Reply:
What if we need a different token in the future? Having the prefix seems useful to have

Reply:
> What if we need a different token in the future? Having the prefix seems useful to have

The trouble here is that 18 means "EAT profile", using the `psa-` prefix is not correct.

Reply:
Sorry, I misread Simon's comment. For some reason I thought he was talking about the URL. No objections to dropping psa- from the comment

Reply:
done in 58ba0c6